src/scripts.js

```diff
@@ -22,20 +25,64 @@ window.addEventListener('load', fetchData())
 function fetchData() {
     Promise.all([getData('customers'), getData('rooms'), getData('bookings')])
     .then(data => {
-        console.log(data)
         allCustomers = data[0].customers
         allRooms = data[1].rooms
         allBookings = data[2].bookings
-        // displayCustomerData(user)
+        currentCustomer = new Customer(allCustomers[Math.floor(Math.random() * allCustomers.length)])
+        displayCustomerData(currentCustomer, allRooms, allBookings)
     })
 }
 
-// function displayCustomerData(user) {
-//     //Hard code user until we have log in page?
-//     //Add user name in header
-//     //Add total cost in header
-//     //Display all past bookings as cards
-//     //Display all upcoming bookings as cards
-// }
+function displayCustomerData(currentCustomer, allRooms, allBookings) {
+    greeting.innerText = `Welcome, ${currentCustomer.name}!`
+    cost.innerHTML = `Your total cost of bookings is $${currentCustomer.getTotalCost(allBookings, allRooms).toFixed(2)}.`
+    let userAllBookings = currentCustomer.getAllBookings(allBookings)
+
+    const date = new Date()
+    let day = date.getDate()
+    let month = date.getMonth() + 1
+    let year = date.getFullYear()
+    let currDate = `${year}/${month}/${day}`
+
+    const userUpcomingBookings = userAllBookings.filter(booking => booking.date >= currDate)
+    userUpcomingBookings.forEach(booking => {
+        allRooms.forEach(room => {
+            if(booking.roomNumber === room.number) {
+                upcomingBookings.innerHTML += 
+                `<article class="booking-card">
+                    <p>Date: ${booking.date}</p>
+                    <p>Room #${booking.roomNumber}</p>
+                    <p>${room.roomType.toUpperCase()}</p>
+                    <p>${room.numBeds} ${room.bedSize.charAt(0).toUpperCase()}${room.bedSize.slice(1)} Bed(s)</p>
+                    <p>Bidet: ${displayBidetStatus(room)}</p>
+                    <p>$${room.costPerNight}</p>
+                </article>`
+            }
+        })
+    })
+
+    const userPastBookings = userAllBookings.filter(booking => booking.date < currDate)
```

Comment:
If you were to add the pastBookings and upcomingBookings to the class you could just iterate over each array here to update the Dom. 
I think it would be fairly simple to move over, it would really just be line 64 as a method in the customer class and you could import the current date from your scripts to the customer class.